src/vs/platform/issue/electron-main/issueMainService.ts

```diff
@@ -173,7 +173,7 @@ export class IssueMainService implements IIssueMainService {
 				});
 
 				this.issueReporterWindow.loadURL(
-					FileAccess.asBrowserUri(`vs/code/electron-sandbox/issue/issueReporter${this.environmentMainService.isBuilt ? '' : '-dev'}.html`).toString(true)
+					FileAccess.asBrowserUri(`vs/workbench/electron-sandbox/issues/issueReporter${this.environmentMainService.isBuilt ? '' : '-dev'}.html`).toString(true)
```

Comment:
I think I would have expected that we can remove any `electron-main` reference from the issue reporter and even use `window.open` trick on desktop. This would ensure we are really using the same solution in both platforms.